src/GitVersionCore/Core/RepositoryMetadataProvider.cs

```diff
@@ -500,6 +500,21 @@ private IEnumerable<BranchCommit> GetMergeCommitsForBranch(Branch branch, Config
             return branchMergeBases;
         }
 
+        public bool HasUncommittedChanges()
```

Comment:
Can this be a property instead of a method?

Reply:
Sure, will do. Implemented it as method as the other aspects of that class were getter methods too. Just meant to "fit in" ^^

Reply:
Ok, I see you're right. I'm not a fan of that sort of Java-ish code, but it's better to be internally consistent than to make an half-hearted attempt at being consistent with C# conventions. So please ignore my request.